crates/primitives/fee/src/lib.rs

```diff
@@ -48,6 +48,12 @@ pub static VM_RESOURCE_FEE_COSTS: [(&str, FixedU128); 7] = [
     ("ec_op_builtin", FixedU128::from_inner(10_240_000_000_000_000_000)),
 ];
 
+pub const TRANSFER_SELECTOR_NAME: &str = "Transfer";
```

Comment:
Is this used anywhere in the code?

Reply:
oh, yes. Forgot about this. Added to tests so that hash from it would be more readable